modules/kafka/kafka-dest-driver.c

```diff
@@ -324,6 +412,32 @@ _purge_remaining_messages(KafkaDestDriver *self)
 
 }
 
+static gboolean
+_init_template_topic_name(KafkaDestDriver *self)
+{
+  self->topicname_is_a_template = TRUE;
+  msg_debug("kafka: The topic name is a template");
```

Comment:
This debug log is either not needed, or should be extended with `log_pipe_location_tag()`, the driver and the template itself.